aks/postgres/variables.tf

```diff
@@ -85,6 +85,16 @@ variable "azure_storage_mb" {
   default = 32768
 }
 
+variable "azure_storage_tier" {
+  type        = string
+  description = "Tier of storage used by the PostgreSQL Flexible Server. Possible values are P4, P6, P10, P15, P20, P30, P40, P50, P60, P70, P80. Defaults to Premium if not specified."
```

Comment:
Not all values are possible, it depends on storage: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/postgresql_flexible_server#storage_tier-defaults-based-on-storage_mb
Could we clarify in the description?